### ISSUE TITLE
Jarvis als Geräte Manager / Devices Adapter

### ISSUE THREAD

**Issue by Zefau** [Zefau/ioBroker.jarvis#1147]
Geräte zentral via Jarvis verwalten und diese in blockly/javascript nutzbar machen.

**Comment by mcuiobroker** [comment]
Jede Komponente (Body-Element, Action-Element usw) von einem DP in einem Gerät hätte dann ein ioBroker-Objekte und der Inhalt könnte dann direkt von ioBroker aus geändert werden? Bei jeder Änderung in ioBroker, würde sich dies dann direkt in allen geöffneten Browsern ändern?

**Comment by Zefau** [comment]
Nein. Geräte aus jarvis sind in blockly nutzbar - als Alternative zum Devices Adapter. Pflege erfolgt in jarvis.
